source/persistence/MessageController.cpp

```diff
@@ -7,6 +8,11 @@
 #include <QDateTime>
 
 #include <QDebug>
+#include <QUrl>
+#include <QDir>
```

Comment:
Where is QDir used?

Reply:
This is used to build the path to the attachment to remove it